lib/RESTService.js

```diff
@@ -95,6 +95,8 @@ export default class RESTService {
 		this.app.put('/recorder/:index/url', (req, res) => this.updateRecorderUrl(req, res));
 		//curl -X PUT -H "Content-Type: application/json" -d '{"baseTopic": "plate-minder"}' localhost:5555/recorder/0/baseTopic
 		this.app.put('/recorder/:index/baseTopic', (req, res) => this.updateRecorderBaseTopic(req, res));
+		//curl -X PUT -H "Content-Type: application/json" -d '{"baseTopic": "plate-minder"}' localhost:5555/recorder/0/enableImage
```

Comment:
Please update and test the curl example request to match what the new path requires.